S/SoapyBladeRF/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg, Base.BinaryPlatforms
+
+name = "SoapyBladeRF"
+version = v"0.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/pothosware/SoapyBladeRF.git", "85f6dc554ed4c618304d99395b19c4e1523675b0")
+]
+
+dependencies = [
+    Dependency("soapysdr_jll"; compat="0.8.0"),
+    Dependency("BladeRFHardwareDriver_jll"; compat="2.4.1")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+mkdir SoapyBladeRF/_build
+cd SoapyBladeRF/_build
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      ..
+
+make -j${nproc}
+make install
+if [[ "${target}" == *-apple-* ]]; then
+    mv ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.so ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.dylib
```

Comment:
1. I recommend using `install -Dvm 0755`
2. Just renaming the file is not enough, the soname is likely wrong

Reply:
1. I'm sorry, I'm not a compile export. Does the first mean I shall replace `make install` with `install -Dvm 0755`?
What is the advantage?
2. To be honest I copied this line from the other SoapySDR bindings:
https://github.com/JuliaPackaging/Yggdrasil/blob/master/S/SoapyUHD/build_tarballs.jl#L32 &
https://github.com/JuliaPackaging/Yggdrasil/blob/master/S/SoapyRTLSDR/build_tarballs.jl#L31 etc

Reply:
Alright I replaced
```
make install
if [[ "${target}" == *-apple-* ]]; then
    mv ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.so ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.dylib
fi
```
with
```
install -Dvm 755 "libbladeRFSupport.so" "${libdir}/SoapySDR/modules0.8/libbladeRFSupport.${dlext}"
```
Is that correct?